acestep/training/dataset_builder_modules/preprocess_vae.py

```
@@ -1,12 +1,150 @@
+    # Auto-select chunk size based on GPU VRAM
+    if chunk_size is None:
+        gpu_mem_gb = 0.0
+        if torch.cuda.is_available():
+            try:
+                props = torch.cuda.get_device_properties(vae_device)
+                gpu_mem_gb = props.total_mem / (1024 ** 3)
+            except Exception:
+                pass
+        chunk_size = _TARGET_SR * 15 if gpu_mem_gb <= 8 else _TARGET_SR * 30
+
+    B, C, S = audio.shape
```

Ruff (0.15.0)

[warning] Unpacked variable `C` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)